RandLAPACK/gpu_functions/rl_cuda_kernels.cuh

```diff
@@ -0,0 +1,934 @@
+// This consitional allows us to make sure that the cuda kernels are only compiled with nvcc.
```

Comment:
Do you mean "conditional"? And are you referring to the ifdef USE_CUDA?

Reply:
Yes, fixed.